frontend/src/components/day-selector/index.tsx

```diff
@@ -32,10 +31,7 @@ export const DaySelector: React.FC<{
         <Select
           sx={{ mt: 3, width: "100%" }}
           onChange={(e: React.ChangeEvent<HTMLSelectElement>) =>
-            router.push(
-              "/[lang]/schedule/[day]",
-              getDayUrl(language, e.target.value),
-            )
+            router.push("/schedule/[day]", getDayUrl(language, e.target.value))
```

Comment:
Do we still need language here??

Reply:
I guess marco left it so he didn't have to change all the places using getDayUrl :D

Reply:
🤔 let me check